### DISCUSSION TITLE
What does this means in madmax settings?

### DISCUSSION THREAD

**Discussion by Spaikys** [ericaltendorf/plotman#889] [Q&A]
What does this means in madmax settings? 
Is it how many plots create or how many plots to create simultaneously?

``` n_rmulti2: 1          # Default is 1  ```

**Comment by altendky** [comment]
The options are pass through.  See the [madMAx readme](https://github.com/madMAx43v3r/chia-plotter#usage).  It is a "multiplier" for the "number of threads" to use in phase 2 (and maybe 3)?  It doesn't say a lot, but here's [an issue discussing it](https://github.com/madMAx43v3r/chia-plotter/issues/809) as well.
